### PR TITLE
Update php.yml

### PR DESCRIPTION
**Pull request by FredoVelcro**
Fix bad escaping of namespaces

### CODE REVIEW

File: symfony2/yaml/php.yml [hunk 1]
```diff
@@ -18,9 +18,9 @@ questions:
         question: 'If function foo() is defined in the namespace myapp\utils\bar and your code is in namespace myapp, what is the correct way to import the bar namespace so you can use the foo() function?'
         answers:
             - {value: "use bar",                 correct: false}
-            - {value: "use myapp\utils\bar",     correct: true}
-            - {value: "use utils\bar",           correct: false}
-            - {value: "use myapp\utils\bar\foo", correct: false}
+            - {value: "use myapp\utils\\bar",     correct: true}
```
Comment (ProPheT777) [inline symfony2/yaml/php.yml]: There is a reason to not escape the first occurrence ?

Comment (ProPheT777) [inline symfony2/yaml/php.yml]: ping @FredoVelcro

**Comment by FredoVelcro** [comment]
Hi,

You're right, all should be escaped !

**Comment by ProPheT777** [comment]
Ok, I wait your change to merge.

**Pull request closed without merging**